contracts/markets/MarketDeployer.sol

```diff
@@ -7,10 +7,10 @@ import "../interfaces/CloberMarketDeployer.sol";
 import "../OrderBook.sol";
 
 contract MarketDeployer is CloberMarketDeployer {
-    address private immutable _factory;
+    address public immutable factory;
```

Comment:
I think this `public` access modifier is not necessary.

Reply:
https://github.com/clober-dex/core/pull/40/commits/5d44d63ff513eaddbf80bc2710112e94d0f7ce80